xml/dm_registers.xml

```diff
@@ -37,6 +28,22 @@
 
             7 (other): An access failed for another reason.
         </field>
+	<field name="0" bits="23" access="R" reset="0" />
+        <field name="impebreak" bits="22" access="R" reset="Preset">
+            If 1, then there is an implicit {\tt ebreak} instruction at the
+            non-existent word immediately after the Program Buffer. This saves
+            the debugger from having to write the {\tt ebreak} itself, and
+            allows the Program Buffer to be one word smaller.
+
+            This must be 1 when \Fprogbufsize is 1.
+        </field>
+	<field name="0" bits="21:20" access="R" reset="0" />
```

Comment:
The diff looks like indenting is not consistent between eg. this line and the previous one.

Reply:
some are tabs and some are spaces. The whole file is inconsistent.

Reply:
:-( Maybe something to fix in a separate PR then.

Reply:
woops I already went through and pushed it to this one.

Reply:
No worries. At least it's fixed now. :-)